src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Crud/Update/Content/_form.html.twig

```diff
@@ -19,7 +16,7 @@
 <div class="page-body">
     <div class="container-xl">
         {% block content %}
-            {% set renderRest = hookable_configuration.render_rest %}
+            {% set renderRest = hookable_metadata.configuration.render_rest %}
```

Comment:
Maybe we should add a global switch in the configuration regarding `render_rest`?